app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by! id: params[:id]
```

Comment:
```suggestion
    @user = User.find_by id: params[:id]
```
xử lý check cho trường hợp nil nữa em nhé